src/plugins/pbjx/components/event-stream/index.jsx

```diff
@@ -81,14 +87,19 @@ class EventStream extends React.Component {
 
   render() {
     const {
+      isEditMode,
+      isRevertGranted,
       getUser,
       response,
       status,
       exception,
+      onRevert: handleRevert,
     } = this.props;
 
     const { allEvents, loadMore } = this.state;
 
+    let index = 0;
```

Comment:
@rsumilang `map` has access to the array index https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map